models/classes/LtiLaunchData.php

```diff
@@ -389,11 +389,20 @@ public function getLaunchLanguage()
      */
     public function getToolConsumerName()
     {
-        return $this->hasVariable(self::TOOL_CONSUMER_INSTANCE_NAME)
-            ? $this->getVariable(self::TOOL_CONSUMER_INSTANCE_NAME)
-            : $this->hasVariable(self::TOOL_CONSUMER_INSTANCE_DESCRIPTION)
-                ? $this->getVariable(self::TOOL_CONSUMER_INSTANCE_DESCRIPTION)
-                : null;
+        $consumerName = null;
+
+        if ($this->hasVariable(self::TOOL_CONSUMER_INSTANCE_NAME)) {
+            $consumerName = $this->getVariable(self::TOOL_CONSUMER_INSTANCE_NAME);
+        }
+
+        if (
+            $consumerName === null
+            && $this->hasVariable(self::TOOL_CONSUMER_INSTANCE_DESCRIPTION)
+        ) {
+            $consumerName = $this->getVariable(self::TOOL_CONSUMER_INSTANCE_DESCRIPTION);
+        }
+
+        return $consumerName;
```

Comment:
Maybe early return to improve readability even more? 

```suggestion

        if ($this->hasVariable(self::TOOL_CONSUMER_INSTANCE_NAME)) {
            return $this->getVariable(self::TOOL_CONSUMER_INSTANCE_NAME);
        }

        if ($this->hasVariable(self::TOOL_CONSUMER_INSTANCE_DESCRIPTION)) {
            return $this->getVariable(self::TOOL_CONSUMER_INSTANCE_DESCRIPTION);
        }

        return null;
```

Reply:
Ok, I agree, if you you are confident that condition with null originally was introduced on purpose, but not because someone was willing to put everything within nested ternaries